packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/telephony-line-phone-accessories.html

```diff
@@ -0,0 +1,36 @@
+<section class="telecom-telephony-line-phone-accessories">
+    <header>
+        <tuc-section-back-link
```

Comment:
could we use a `oui-back-button` instead ? It is the way it is done elsewhere (in other universes) and it will be easier to update and provide improvements if the same component is used everywhere